test/example_data.dart

```diff
@@ -147,6 +147,19 @@ final User fourthUser  = user(fullName: 'Fourth User', email: 'fourth@example');
 // Streams and subscriptions.
 //
 
+/// A fresh stream ID, from a random but always strictly increasing sequence.
+int _nextStreamId() => (_lastStreamId += 1 + Random().nextInt(10));
+int _lastStreamId = 100;
+
+const staticStreamId = 123;
```

Comment:
Or I guess that name doesn't capture it either — the meaning is actually a little complicated, in that this is the default for `eg.streamMessage` but not for `eg.stream` (which defaults to the random sequence).

Let's make it:
```suggestion
const defaultStreamMessageStreamId = 123;
```

and also put the definition right above `streamMessage`.

That name is kind of wordy, but that may actually be a good thing here — the way this works is a bit surprising and we should probably reorganize so that tests aren't referring to a shared constant stream ID like this.

The `streamMessage` doc should also mention that there's this default.

Reply:
bump on these two aspects:

> and also put the definition right above `streamMessage`.

> The `streamMessage` doc should also mention that there's this default.